ibm_catalog.json

```diff
@@ -57,6 +57,115 @@
               }
             ]
           },
+          "configuration" : [
+            {
+              "key": "ibmcloud_api_key"
+            },
+            {
+              "key": "use_existing_resource_group"
+            },
+            {
+              "key": "resource_group_name"
+            },
+            {
+              "custom_config": {
+                "config_constraints": {
+                  "generationType": "2"
+                },
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "type": "region"
+              },
+              "key": "region",
+              "required": true,
+              "type": "string"
+            },
+            {
+              "key": "secrets_manager_instance_name"
+            },
+            {
+              "key": "service_plan",
+              "options": [
+                {
+                  "displayname": "Standard",
+                  "value": "standard"
+                },
+                {
+                  "displayname": "Trial",
+                  "value": "trial"
+                }
+              ]
+            },
+            {
+              "key": "allowed_network",
+              "options": [
+                {
+                  "displayname": "Public and private",
+                  "value": "public-and-private"
+                },
+                {
+                  "displayname": "Private only",
+                  "value": "private-only"
+                }
+              ]
+            },
+            {
+              "key": "secret_manager_tags"
+            },
+            {
+              "key": "iam_engine_enabled"
+            },
+            {
+              "key": "iam_engine_name"
+            },
+            {
+              "key": "skip_kms_iam_authorization_policy"
+            },
+            {
+              "key": "existing_sm_kms_key_crn"
```

Comment:
this has been renamed to `existing_secrets_manager_kms_key_crn` in a recent PR - I suggest you review what is in main branch now